frontend/src/App.vue

```diff
@@ -1,32 +1,48 @@
+
 <template>
-  <v-app>
-    <v-toolbar app>
-      <v-toolbar-title class="headline text-uppercase">
-        <span>Vuetify</span>
-        <span class="font-weight-light">MATERIAL DESIGN</span>
-      </v-toolbar-title>
-      <v-spacer></v-spacer>
-      <v-btn
-        flat
-        href="https://github.com/vuetifyjs/vuetify/releases/latest"
-        target="_blank"
-      >
-        <span class="mr-2">Latest Release</span>
-        <v-icon>open_in_new</v-icon>
-      </v-btn>
+    <v-app>
+    <v-toolbar>
+        <v-toolbar-title class="headline text-uppercase">
+        <span>Amazon Dash</span>
+        </v-toolbar-title>
     </v-toolbar>
-
-    <v-content>
-      <router-view/>
+    <v-content app>
+    <div class="container">
+        <div class="row">
+            <div class="col-md-7">
+                <div class="row">
+                    <div :key="service.id" class="col-md-6" v-for="service in services">
+                          <service :service="service"></service>
+                    </div>
+                </div>
+            </div>
+            <div class="col-md-5"></div>
+        </div>
+    </div>
     </v-content>
-  </v-app>
+    </v-app>
 </template>
 
-<script lang="ts">
-import { Component, Vue } from 'vue-property-decorator'
+<script>
+
+
+import services from '@/services.json'
+import Service from '@/components/Service.vue';
 
-@Component
-export default class App extends Vue {
 
+export default {
```

Comment:
Please look at `src/views/Home.vue for an example of how to set this up using TypeScript. Then redo that in `src/views/Services.vue`